From: Teemu Hukkanen <tjhukkan@iki.fi>
Date: Thu, 3 Aug 2023 02:43:04 +0300
Subject: Use joystick identifier from SDL_JoystickOpen for SDL_JoystickName

---
 src/gui/gamepad.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gui/gamepad.cpp b/src/gui/gamepad.cpp
index a7a9498..f72cc0c 100644
--- a/src/gui/gamepad.cpp
+++ b/src/gui/gamepad.cpp
@@ -55,7 +55,7 @@ void Gamepad::AllocateJoysticks(void)
 		// We need to copy the contents of this pointer, as SDL will change it
 		// willy nilly to suit itself
 //		padName[i] = SDL_JoystickName(i);
-		strncpy(padName[i], SDL_JoystickName(i), 127);
+		strncpy(padName[i], SDL_JoystickName(pad[i]), 127);
 		padName[i][127] = 0;	// Just in case name's length > 127
 
 		if (pad[i])
