Description: Add kFreeBSD support
Author: Teemu Hukkanen <tjhukkan@iki.fi>

--- virtualjaguar-2.1.0.orig/jaguarcore.mak
+++ virtualjaguar-2.1.0/jaguarcore.mak
@@ -33,6 +33,11 @@ else ifeq "$(findstring Linux,$(OSTYPE))
 SYSTYPE    := __GCCUNIX__
 SDLLIBTYPE := --libs
 
+else ifeq "$(findstring kFreeBSD,$(OSTYPE))" "kFreeBSD"
+
+SYSTYPE    := __GCCUNIX__
+SDLLIBTYPE := --libs
+
 # ??? Throw error, unknown OS
 else
 
